### ISSUE TITLE
add conversion of decibels to microwatts

### ISSUE THREAD

**Issue by OmlineEditor** [ferraridamiano/ConverterNOW#113]
please add conversion of decibels to microwatts and back.
example by link: https://www.rapidtables.org/ru/convert/power/dBm_to_mW.html

**Comment by ferraridamiano** [comment]
What are you talking about is [decibal watt](https://en.wikipedia.org/wiki/Decibel_watt) I think.
I will add it, but I need to do some change in the [library](https://github.com/ferraridamiano/units_converter) that I use because at the moment it doesn't support logarithmic relation between units.

**Comment by OmlineEditor** [comment]
That's right. you gave the correct link to wikipedia. if possible, make a new converter.
